src/components/PurchaseSummary/PurchaseSummaryAccordion/style.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'
+import { Colors, mediaQueries } from '../../../tokens'
+
+export const AccordionContainer = styled.div`
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 14px;
+`
+
+export const AccordionHeader = styled.div`
+  width: 100%;
+  background-color: ${Colors.light.neutral.neutral100};
```

Comment:
In all cases where we deal with color we need to

Reply:
All comments were resolved, you can see here -  6fd9bf154071d5db476f92d21cd3e884ce29d38b